### PR TITLE
feat(route): add LiSA

### PR DESCRIPTION
**Pull request by Kiotlin**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/lxixsxa/info
/lxixsxa/disco
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note
LiSA is a singer from Japan, these routes have implemented the rss feeds for both news and latest discography.

### CODE REVIEW

File: docs/en/live.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ pageClass: routes
...
 
+## LiSA
+
+### News
+
+<RouteEn author="Kiotlin" example="/lisa/info" path="/lisa/info" radar="1" rssbud="1" />
+
+### Latest Discography
+
+<RouteEn author="Kiotlin" example="/lisa/disco" path="/lisa/disco" radar="1" rssbud="1" />
```
Comment (TonyRL) [inline docs/en/live.md:15]:
```suggestion
<RouteEn author="Kiotlin" example="/lisa/disco" path="/lisa/disco" radar="1" rssbud="1" />

```
Missing a linebreak

File: lib/v2/lisa/discography.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        // iterate through all leaf objects
+        item: data.map((item) => ({
+            // the article title
+            title: item.title,
+            // the article content
+            description: `
+                <div>${item.comment} Type: ${item.type} Price: ${item.price}</div>
+                <img src="${item.imageLink}"><br>
+                ${item.description}
+            `,
```
Comment (TonyRL) [inline lib/v2/lisa/discography.js:43]: Please use `art-template`
docs: https://docs-rsshub.pages.dev/en/joinus/script-standard.html#template

File: lib/v2/lisa/information.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        description: "Let's see what is new about LiSA.",
+        // iterate through all leaf objects
+        item: data.map((item) => ({
+            // the article title
+            title: item.title,
+            // the article content
+            description: `
+                <span>Category: ${item.category}</span><br>
+                <div>${item.description}</div>
+            `,
```
Comment (TonyRL) [inline lib/v2/lisa/information.js:39]: Please use `art-template`
docs: https://docs-rsshub.pages.dev/en/joinus/script-standard.html#template

File: lib/v2/lisa/radar.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                source: ['/artist/lisa/'],
+                target: '/lisa/disco',
+            },
+        ],
+    },
+    'lxixsxa.com': {
+        _name: 'LiSA Official',
+        www: [
+            {
+                title: 'LiSA News',
```
Comment (TonyRL) [inline lib/v2/lisa/radar.js:23]:
```suggestion
                title: 'News',
```
title should match H3 heading of the docs

File: lib/v2/lisa/radar.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+                docs: 'https://docs.rsshub.app/other.html#lisa',
```
Comment (TonyRL) [inline lib/v2/lisa/radar.js:24]: It should point to `/other.html`

**Review by TonyRL — COMMENTED** [review]
Please use `/lxixsxa/info` instead as the domain name is `lxixsxa.com`

File: lib/v2/lisa/radar.js (hunk 5 above)
Comment (Kiotlin) [reply lib/v2/lisa/radar.js:24]: Thank you for your review.
I have mistaken all the docs link, I will change them all to `https://docs.rsshub.app/en/live.html#lisa`, which is the doc page url of lisa router when being merged. Do I fix them correctly?

Comment (TonyRL) [reply lib/v2/lisa/radar.js:24]: Yes

**Comment by Kiotlin** [comment]
> Please use `/lxixsxa/info` instead as the domain name is `lxixsxa.com`

You mean it's better to change the router directory name from 'lisa' to 'lxixsxa'?

But actually this router get info from sony artisit page, and will be detected when browing sony artist page and lisa's official page(lxixsxa.com), which both refer to artist 'LiSA'. I wonder if other people would know what lxixsxa exactly is, cause most people only heard about artist's name, but not official site's domain name...

**Comment by Kiotlin** [comment]
All reviews fixed.

**Comment by Kiotlin** [comment]
By the way, may i ask a question about jsonp handling? Is there a better solution to handle jsonp with got in RSSHub （I have checked the document of got, didn't find any.）. What other guys usually do when getting into this?

In this router I implement a function to convert jsonp to json, which makes me feel a little bit reluctant. Are there any more 'elegant' solution or advice? Thanks a lot!

**Comment by TonyRL** [comment]
> You mean it's better to change the router directory name from 'lisa' to 'lxixsxa'?

Yes

> But actually this router get info from sony artisit page, ...

From what I've observed, the official page(lxixsxa.com) grabs data from sony's site.

> I wonder if other people would know what lxixsxa exactly is, cause most people only heard about artist's name, but not official site's domain name

I guess people will look up the docs instead of brute-forcing the route. You don't need to change the H2 heading.

> Is there a better solution to handle jsonp

Either manipulate the string manually or find a package (like [jsonp](https://www.npmjs.com/package/jsonp)) that did that for you and `npm install`

**Comment by Kiotlin** [comment]
Fixed. Thanks!

File: lib/v2/lxixsxa/templates/disco.art [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<span>{{comment}} Type: {{type}} Price: {{price}}</span>
+{{ if image }}
+    <img src={{image}} />
+{{ /if }}
+{{ description }}
```
Comment (TonyRL) [inline lib/v2/lxixsxa/templates/disco.art:5]:
```suggestion
{{@ description }}
```

File: lib/v2/lxixsxa/templates/info.art [hunk 7]
```diff
@@ -0,0 +1,4 @@
+{{ if category }}
+    <span>Category: {{category}}</span>
+{{ /if }}
+{{ description }}
```
Comment (TonyRL) [inline lib/v2/lxixsxa/templates/info.art:4]:
```suggestion
{{@ description }}
```

**Pull request merged**